### ISSUE TITLE
Device detection not working in Safari Responsive Design Mode

### ISSUE THREAD

**Issue by JurgenJocubeit** [fnando/browser#353]
## Description

Browser device not being detected correctly in Safari version 11.1 (13605.1.33.1.4) when using Responsive Design Mode. Same code works fine in Chrome when using Device Toggle feature in developer tools.

## How to reproduce

Using above Safari version choose Responsive Design Mode from the Develop menu. Switch devices and re-render a page which reports the browser device.

## What do you expect

Expect `browser.device.mobile?` and `browser.device.tablet?` to report true when changing device in Response Design Mode.

## What happened instead

`browser.device.desktop?` returns true regardless of device and UA selected.

## Software:

- Rails version: 5.2
- Browser gem version: 2.5.3

**Comment by lowjarp** [comment]
I am having the same issue.. but only after the update to mojave..

**Comment by fnando** [comment]
You have to reload the page to change the user agent. Simply clicking the different viewport modes won't do it.
